CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+- Update Metrics Server to v2.11.2 ([#1116](https://github.com/kinvolk/lokomotive/pull/1116)).
+- Update Cluster Autoscaler to version v1.1.0 ([#1137](https://github.com/kinvolk/lokomotive/pull/1137)). Release info:
+- Update Rook to v1.4.6 ([#1117](https://github.com/kinvolk/lokomotive/pull/1117)).
+- Update Velero to v1.5.2 ([#1131](https://github.com/kinvolk/lokomotive/pull/1131)).
+- Update OpenEBS to v2.2.0 ([#1095](https://github.com/kinvolk/lokomotive/pull/1095)).
+- Update Contour to v1.10.0 ([#1170](https://github.com/kinvolk/lokomotive/pull/1170).
+- Update Linkerd to stable-2.9.0 ([#1123](https://github.com/kinvolk/lokomotive/pull/1123)).
+- Update Web UI to v0.1.3 ([#1237](https://github.com/kinvolk/lokomotive/pull/1237)).
+- Update Prometheus Operator to v0.43.2 ([#1162](https://github.com/kinvolk/lokomotive/pull/1162)).
+- Update Calico to v3.17.0 ([#1251](https://github.com/kinvolk/lokomotive/pull/1251)).
+
+#### Terraform provider updates
+
+- Update Terraform providers to their latest versions([#1133](https://github.com/kinvolk/lokomotive/pull/1133)).
+
+#### Other updates
+
+- Update aws-ebs-csi-driver to v0.7.0 ([#1135](https://github.com/kinvolk/lokomotive/pull/1135)).
+
+#### New platforms
+
+- Add support for Tinkerbell as a platform ([#392](https://github.com/kinvolk/lokomotive/pull/392)).
+
+#### Bug fixes
+
+- Fix bug adding worker pools with TLS bootstrap enabled ([#1181](https://github.com/kinvolk/lokomotive/pull/1181)).
+- contour: apply node affinity and tolerations also to certgen job ([#1161](https://github.com/kinvolk/lokomotive/pull/1161)).
+- Improve hardware reservations validation rules on Equinix Metal ([#1186](https://github.com/kinvolk/lokomotive/pull/1186)).
+- Equinix Metal: ensure Calico CRDs gets created before use ([#1177](https://github.com/kinvolk/lokomotive/pull/1177)). They were failing occasionally.
+- openebs-operator: set webhook policy to Ignore ([#1172](https://github.com/kinvolk/lokomotive/pull/1172)). This works around a bug in Kubernetes, see this [comment](https://github.com/openebs/openebs/issues/3046#issuecomment-642558701) for details.
+- Don't run control plane components as DaemonSets on single control plane node clusters ([#1193](https://github.com/kinvolk/lokomotive/pull/1193)).
+- calico-host-protection: Add custom locked down PSP configuration ([#1274](https://github.com/kinvolk/lokomotive/pull/1274)).
+
+#### Features
+
+- Add Packet CCM to Packet platform ([#1155](https://github.com/kinvolk/lokomotive/pull/1155)).
+- contour: parameterize Envoy scraping interval ([#1229](https://github.com/kinvolk/lokomotive/pull/1229)).
+- Expose --conntrack-max-per-core kube-proxy flag ([#1187](https://github.com/kinvolk/lokomotive/pull/1187)).
+- Add `require_volume_annotation` for restic plugin ([#1132](https://github.com/kinvolk/lokomotive/pull/1132)). This PR is based on top of #1131 and would need to be merged first.
```

Comment:
```suggestion
- Add `require_volume_annotation` for restic plugin ([#1132](https://github.com/kinvolk/lokomotive/pull/1132)).
```

I'm not sure the last sentence belongs in the release notes. Is there an open TODO item there? What is the context here?

Reply:
I think it was a left over from formatting the release script output.